### PR TITLE
Nova: add overrides suse images

### PR DESCRIPTION
**Pull request by Itxaka**
Provides the proper overrides in the nova template for helm

### CODE REVIEW

**Comment by Itxaka** [comment]
not merge until #178 gets in and we rework the templates

**Comment by evrardjp** [comment]
No need to pull a patch here?

**Comment by Itxaka** [comment]
> No need to pull a patch here?

already in the list of patches (642067) :)

File: playbooks/roles/deploy-osh/templates/nova.yaml.j2 [hunk 1]
```diff
@@ -6,4 +6,50 @@ conf:
       user: {{ ses_cluster_configuration['cinder']['rbd_store_user'] }}
       keyring: {{ suse_ses_cinder_pool_key | default(ceph_admin_keyring_key) }}
       secret_uuid: {{ libvirt_ceph_cinder_secret_uuid }}
-{{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
+{% if developer_mode %}
+  apache2:
+    binary: apache2ctl
+    start_parameters: -DFOREGROUND -k start
+    site_dir: /etc/apache2/vhosts.d
+    conf_dir: /etc/apache2/conf.d
```
Comment (jsuchome) [inline playbooks/roles/deploy-osh/templates/nova.yaml.j2]: I think you need version module here as well

Comment (Itxaka) [reply playbooks/roles/deploy-osh/templates/nova.yaml.j2]: that is what I thougth but someone this does not fail...

I do not understand apache sometimes

**Comment by evrardjp** [comment]
642067 doesn't cleanly apply, can we update the patch in here?

**Comment by jsuchome** [comment]
@evrardjp I think that's because the cell setup has not been executed, so we have to look at other errors

**Comment by Itxaka** [comment]
The errors are probably coming from the nova-cell not being executed as nova-scheduler is running but not ready.

That should be fixed by https://review.openstack.org/#/c/644907/ let me add it to this patch

**Comment by evrardjp** [comment]
Conflict in dev patcher. I am solving it locally and I will see how it goes.

**Comment by jsuchome** [comment]
That's the new patch for probes,   644907  I guess. Probably needs a rebase after those probes changes were merged

**Comment by evrardjp** [comment]
> That's the new patch for probes, 644907 I guess. Probably needs a rebase after those probes changes were merged
> 
> UPDATE: oh, now I see _this_ PR has conflicts...

Yes, sorry, that was unclear :) This PR has conflicts on the level of the addition of 644907 in dev patcher on top of 645189. I am trying to run with both applied, with or without libvirt, and it always fails for me.

**Comment by Itxaka** [comment]
rebased!

**Comment by evrardjp** [comment]
weird issue on my side during the helm testing. The deploy went fine, and the pods look okay.

**Pull request merged**